tests/integrated/test-i18n/test-i18n.cxx

```diff
@@ -0,0 +1,19 @@
+
+#include "bout.hxx"
+#include "bout/sys/gettext.hxx"
+
+int main(int argc, char**argv) {
+  // Setting the i18n environment
+  if (setlocale (LC_ALL, "") == NULL) {
+    throw BoutException("Failed to set locale");
+  }
+  bindtextdomain ("test", "/home/bd512/BOUT-dev/locale/");
```

Comment:
I guess this won't work for anyone that isn't you!

Reply:
Ah yes, this "test" was really just for while I was working out how to use `gettext`. It needs changing into an actual test. The `configure` and `BoutInitialise` calls use a portable method to set the path.